src/main/java/org/apache/fineract/operations/BatchDTO.java

```diff
@@ -54,11 +54,13 @@ public class BatchDTO {
     private String payerFsp;
 
     private String correlationId;
+    private Long approvedAmount;
```

Comment:
There is a Batch DTO file in Bulk processor,  approvedAmount and approvedCount to be added there.